app/scripts/models/email-resend.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This model limits the number of emails a component can send
+
+'use strict';
+
+define([
+  'backbone',
+  'lib/constants'
+], function (Backbone, Constants) {
+
+  function _shouldResend (tries) {
+    return tries === 1 || tries === 4;
+  }
+
+  var EmailResend = Backbone.Model.extend({
+    defaults: {
+      tries: 0,
+      maxTries: Constants.EMAIL_RESEND_MAX_TRIES,
+      onTriesExceeded: undefined,
+      shouldResend: _shouldResend
+    },
+
+    requestResend: function () {
+      var tries = this.get('tries') + 1;
+      this.set('tries', tries);
+
+      if (tries === this.get('maxTries') + 1) {
+        var onTriesExceeded = this.get('onTriesExceeded');
```

Comment:
Again, instead of storing `onTriesExceeded` just do a simple:

```js
this.trigger('maxTriesExceeded');
```

The creator/consumer of the model can do a:
```js
emailResendModel.on('maxTriesExceeded', handleMaxTries);
```